Xamarin.Forms.Core.UnitTests/StepperUnitTests.cs

```diff
@@ -160,17 +163,35 @@ public void StepperValueChangedEventArgs (double initialValue, double finalValue
 			var oldValue = 0.0;
 			var newValue = 0.0;
 
-			stepper.ValueChanged += (s, e) => {
+			stepper.ValueChanged += (s, e) =>
+			{
 				stepperFromSender = (Stepper)s;
 				oldValue = e.OldValue;
 				newValue = e.NewValue;
 			};
 
 			stepper.Value = finalValue;
 
-			Assert.AreEqual (stepper, stepperFromSender);
-			Assert.AreEqual (initialValue, oldValue);
-			Assert.AreEqual (finalValue, newValue);
+			Assert.AreEqual(stepper, stepperFromSender);
+			Assert.AreEqual(initialValue, oldValue);
+			Assert.AreEqual(finalValue, newValue);
+		}
+
+		[TestCase(10)]
+		public void TestReturnToZero(int steps)
+		{
+			var stepper = new Stepper(0, 10, 0, 0.5);
```

Comment:
this test is passing without the change (.5 has an exact fp representation)

Reply:
Wasn't the issue reported that it was numbers less than 0.5 and with large Min/Max values, these tests are not testing that case?
I think the issue was reported with 
Min = -100000
Max = 100000
Step = Any value less than 0.5

Reply:
don't bother, those are notes to myself, and history. I'm handling this. But I reverted your code, and this test was still passing.

Reply:
> Wasn't the issue reported that it was numbers less than 0.5 and with large Min/Max values, these tests are not testing that case?
> I think the issue was reported with
> Min = -100000
> Max = 100000
> Step = Any value less than 0.5

step value range was between -1 and 1 and larg min and max values